deploy/examples/create-external-cluster-resources.py

```diff
@@ -1793,6 +1794,7 @@ def upgrade_users_permissions(self):
             "client.csi-cephfs-provisioner",
             "client.csi-rbd-node",
             "client.csi-rbd-provisioner",
+            "client.healthchecker",
```

Comment:
You need to update the     `def get_caps_and_entity(self, user_name):`
and return the client and default caps of health checker,

Similar to what we have for other users,